### ISSUE TITLE
No such file or directory: 'task1_test_split.txt' and supp_similarity_8.pkl'

### ISSUE THREAD

**Issue by one23sunnyQQ** [RyanXLi/OneshotDet#3]
Hi, I tried to train using command from readme. But,  get these errors:
        No such file or directory: 'task1_test_split.txt' and supp_similarity_8.pkl'

**Comment by MagnetoYuWang** [comment]
> Hi, I tried to train using command from readme. But, get these errors: No such file or directory: 'task1_test_split.txt' and supp_similarity_8.pkl'

Yes, I was stuck with this problem too. The author didn't provide this file, how did you solve it? Any other anchor-free repository based recommendations?

**Comment by raeony** [comment]
> > Hi, I tried to train using command from readme. But, get these errors: No such file or directory: 'task1_test_split.txt' and supp_similarity_8.pkl'
> 
> Yes, I was stuck with this problem too. The author didn't provide this file, how did you solve it? Any other anchor-free repository based recommendations?

hello，I also encountered the same problem, may i ask if you solved it, i really want to figure it out, thank you
